examples/bri-3/src/bri/state/api/state.controller.ts

```diff
@@ -0,0 +1,15 @@
+import { Controller, Get, Query } from '@nestjs/common';
+import { QueryBus } from '@nestjs/cqrs';
+import { GetStateContentQuery } from '../capabilities/getStateContent/getStateContent.query';
+
+@Controller('state')
+export class StateController {
+  constructor(private queryBus: QueryBus) {}
+
+  @Get('/')
+  async getLeaveValueContent(
```

Comment:
getLeafValueContent

Reply:
change naming scheme for all to stateTreeLeafValueContent